src/webviews/webview-side/common/main.ts

```diff
@@ -1,7 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-/* eslint-disable @typescript-eslint/no-unused-vars */
+export {};
```

Comment:
I believe this lets webpack build with the `declare __webpack_public_path` there.